less/modalOverlay.less

```diff
@@ -6,6 +6,7 @@
 
 .modal {
   opacity: 1;
+  overflow: scroll;
```

Comment:
I don't think you always want to show a scrollbar on windows here and on mac depending on the setting.  I think you want overflow: auto right?